corehq/apps/users/views/__init__.py

```diff
@@ -461,10 +461,17 @@ def post(self, request, *args, **kwargs):
                 )
 
         if self.request.POST['form_type'] == "update-user-permissions" and self.can_grant_superuser_access:
-            is_super_user = True if 'super_user' in self.request.POST and self.request.POST['super_user'] == 'on' else False
+            is_super_user = 'super_user' in self.request.POST and self.request.POST['super_user'] == 'on'
```

Comment:
You didn't introduce this, but it'd be marvelous if the same function didn't use both `superuser` and `super_user`.

Reply:
https://github.com/dimagi/commcare-hq/pull/30034/commits/9e36dd1d4ab51edc52644f35512a8f88277b21c9